calib3d.go

```diff
@@ -67,6 +67,24 @@ func FisheyeUndistortImageWithParams(distorted Mat, undistorted *Mat, k, d, knew
 	C.Fisheye_UndistortImageWithParams(distorted.Ptr(), undistorted.Ptr(), k.Ptr(), d.Ptr(), knew.Ptr(), sz)
 }
 
+// FisheyeUndistortPoints transforms points to compensate for fisheye lens distortion
+func FisheyeUndistortPoints(distorted Mat, undistorted *Mat, k, d, r, p Mat) {
+	C.Fisheye_UndistortPoints(distorted.Ptr(), undistorted.Ptr(), k.Ptr(), d.Ptr(), r.Ptr(), p.Ptr())
+}
+
+// EstimateNewCameraMatrixForUndistortRectify estimates new camera matrix for undistortion or rectification.
```

Comment:
Perhaps you can add the info here linking to the OpenCV docs for this function?

https://docs.opencv.org/master/db/d58/group__calib3d__fisheye.html#ga384940fdf04c03e362e94b6eb9b673c9